rest-service/manager_rest/resource_manager.py

```diff
@@ -580,6 +583,37 @@ def delete_deployment(self,
         # Delete deployment data  DB (should only happen AFTER the workflow
         # finished successfully, hence the delete_db_mode flag)
         else:
+            # check for external targets
+            deployment_dependencies = self.sm.list(
+                models.InterDeploymentDependencies,
+                filters={'source_deployment': deployment})
+            external_targets = set(
+                json.dumps(x.external_target) for x in
+                deployment_dependencies if x.external_target)
+            if not external_targets:
+                return self.sm.delete(deployment)
+
+            # if external targets found - delete them from IDD in ext. managers
+            manager_ips = [x.private_ip for x in self.sm.list(models.Manager)]
```

Comment:
Same comment as above, x should be informative.

Reply:
1